sensor/common/scannerclient/grpc_client.go

```diff
@@ -23,6 +23,14 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+const (
+	// v4IndexerVersion represents an arbitrary indexer version, at this time
+	// a non-empty value will be interpreted as a Scanner V4 index by Central.
+	//
+	// TODO(ROX-21362): Replace this with the actual version from the indexer API.
```

Comment:
https://github.com/stackrox/stackrox/pull/9355

Reply:
Nice, since https://github.com/stackrox/stackrox/pull/9355 is not merged will keep this here